LetsDinnerV2 MessagesExtension/Controller/CustomRecipeDetailsVC/CustomRecipeDetailsViewModel.swift

```diff
@@ -35,6 +35,7 @@ class CustomRecipeDetailsViewModel {
                     switch result {
                     case .success():
                         self.deleteRecipeObserver.send(value: ())
+                        
```

Comment:
Why adding a blank space?

Reply:
we are not going to abandon customRecipeDetailsVC